src/pages/graphql/payment-methods/payment-services-checkout.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Payment Services as a payment method
+description: Learn how to use the GraphQL API mutations for the Payment Services solution.
+keywords:
+  - GraphQL
+  - Payments
+---
+
+# Payment services as a payment method
+
+Payment Services for Adobe Commerce and Magento Open Source is your turnkey self-service solution, including sandbox testing and a simple setup, for providing robust and secure payment processing for your Commerce websites.
+
+## Payment services checkout workflow
+
+![Payment Services sequence diagram](../../_images/graphql/payment-services.svg)
+
+These steps describe the additional flow of requests and responses required to complete a typical [checkout](../tutorials/checkout/index.md) authorization with [Payment Services](https://experienceleague.adobe.com/docs/commerce-merchant-services/payment-services/guide-overview.html){target=_blank} solution enabled.
+
+1. Fetch the payment configuration by running the `getPaymentConfig` mutation in order to render payment methods.
+
+1. Adobe Commerce returns available payment configuration.
+
+1. Use the `setPaymentMethodOnCart` mutation to [set the payment method](../tutorials/checkout/set-payment-method.md) for the order.
+
+1. Adobe Commerce returns the `cart_id` with the available payment methods for the order.
+
+1. Creates the order with the `createPaymentOrder` mutation.
+
+1. Request forwarded to PayPal.
+
+1. PayPal returns `order_id` and `mp_order_id` values.
+
+1. Adobe Commerce forwards values to client.
+
+1. (Optiona) Run the `getPaymentOrder` mutation if hosted fields and the Signifyd integration are enabled.
+
+1. Adobe Commerce returns these details.
```

Comment:
I asked Thomas about this step. We run the query, but the response isn't a part of the `placeOrder` mutation.

Reply:
done